### PR TITLE
[iOS] Feature/add cards screen

### PR DESCRIPTION
**Pull request by taraspasichnyk**

### CODE REVIEW

**Review by andrii-puhach — DISMISSED** [review]

File: iosApp/iosApp/View/AnswerCard.swift [hunk 1]
```diff
@@ -9,40 +9,55 @@
 import SwiftUI
 
 struct AnswerCard: View {
+
     let answer: String
+    let font: Font
```
Comment (artem-y-eleks) [inline iosApp/iosApp/View/AnswerCard.swift:14]: Not critical, but I'd suggest using a modifier `.font` modifier instead, applied to `AnswerCard`.  
Since you're passing it anyway, this would look more natural in SwiftUI:  
```
AnswerCard(answer: "Some answer")
    .font(.inputPrimary)
```

File: iosApp/iosApp/View/AnswerCard.swift [hunk 2]
```diff
@@ -9,40 +9,55 @@
...
-                    .padding(.top, 20)
-                    .padding(.horizontal, .medium)
-                Spacer()
-            }
-                .font(.inputPrimary)
+        VStack {
+            Text(answer)
+                .frame(maxWidth: .infinity)
+                .multilineTextAlignment(.center)
+                .padding(.top, 16.0)
```
Comment (artem-y-eleks) [inline iosApp/iosApp/View/AnswerCard.swift:23]: We can use `Size` for padding here:
```suggestion
                .padding(.top, .medium)
```

File: iosApp/iosApp/View/Cards/CardsView.swift [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+extension CardItem: Equatable {
+    static func ==(lhs: CardItem, rhs: CardItem) -> Bool {
+        return lhs.id == rhs.id && lhs.id == rhs.id
+    }
+}
+
+import shared
+
+class CardItemsDataModel: ObservableObject {
```
Comment (artem-y-eleks) [inline iosApp/iosApp/View/Cards/CardsView.swift:82]: I know these are related to the `CardsView`, but how about moving them out into a separate file? And we can maybe just group them in a folder if needed

File: iosApp/iosApp/View/Lobby/LobbyView.swift [hunk 4]
```diff
@@ -14,7 +14,7 @@ struct LobbyView: View {
     @EnvironmentObject private var loadingState: LoadingState
 
     @State private var users: [Player] = []
-    @State private var roomCode: String = ""
+    @State private var roomCode: String = "000000"
```
Comment (artem-y-eleks) [inline iosApp/iosApp/View/Lobby/LobbyView.swift:17]: Not sure why is this necessary?

File: iosApp/iosApp/View/Cards/CardsView.swift (hunk 3 above)
Comment (taraspasichnyk) [reply iosApp/iosApp/View/Cards/CardsView.swift:82]: It is probably mot going to exist in next PRs. I am going to start connecting `GameViewModel` and move to a separate files after that. Ignore most of the `CardView` for now.

File: iosApp/iosApp/View/Lobby/LobbyView.swift (hunk 4 above)
Comment (taraspasichnyk) [reply iosApp/iosApp/View/Lobby/LobbyView.swift:17]: To display a preview that contains a default value instead of empty space. It is still being overriden on launch.

**Review by artem-y-eleks — APPROVED** [review]

**Review by artem-y-eleks — APPROVED** [review]
